javascript/javascript2/week1/index2.html

```diff
@@ -0,0 +1,14 @@
+<head>
```

Comment:
Don't forget to use the Doctype/HTML tag.
A nice trick is using the shortcut typing "!" on your VScode. 
Try it on 😉